apps/website/src/pages/docs/dialog.mdx

```diff
@@ -79,6 +79,17 @@ The two buttons at the bottom right are always pinned in that same area, regardl
 
 As specified earlier, full-page dialogs will house various settings options as shown in the example below.
 
+### Placement
+
+A placement dialog can be adjusted to sit in one of the four corners of the page.
+
+<LiveExample src='Dialog.placement.tsx'>
+  <AllExamples.DialogPlacementExample client:load />
+</LiveExample>
+
+Positions for the dialog on the page include:
+`top-left`, `top-right`, `bottom-left`, `bottom-right`
```

Comment:
nit: phrasing improvement (there is no such thing as a "placement dialog", only a `placement` prop)
```suggestion
Using the `placement` prop, a dialog can be adjusted to sit in one of the four corners of the page. Accepted values:
- `top-left`
- `top-right`
- `bottom-left`
- `bottom-right`

<LiveExample src='Dialog.placement.tsx'>
  <AllExamples.DialogPlacementExample client:load />
</LiveExample>
```

Reply:
after accepting the suggestion on github, can you also format the file (by saving it locally)?